tools/packaging/kata-deploy/binary/Cargo.toml

```diff
@@ -0,0 +1,53 @@
+[package]
+name = "kata-deploy"
+version = "0.1.0"
+edition = "2021"
+rust-version = "1.88.0"
```

Comment:
I'd like to know about why choose the rust-1.88.0 but not 1.85.1 which is recommanded in version.yaml ?